float64 x_measurement
float64 y_measurement
float64 z_measurement
float64 pitch_measurement
float64 yaw_measurement
float64 x_input
float64 y_input
float64 z_input
float64 pitch_input
float64 yaw_input
float64 x_pred
float64 y_pred
float64 z_pred
float64 pitch_pred
float64 yaw_pred
float64 x_est
float64 y_est
float64 z_est
float64 pitch_est
float64 yaw_est
